src/card.php

```diff
@@ -2,6 +2,36 @@
 
 declare(strict_types=1);
 
+/**
+ * Set headers and echo response based on type
+ *
+ * @param string|array $output
+ * @return void
+ */
+function renderOutput(string|array $output): void
+{
+    $requestedType = $_REQUEST['type'] ?? 'svg';
+
+    $card = gettype($output) === "string" ? generateErrorCard($output) : generateCard($output);
+
+    if ($requestedType === "json") {
+        $data = gettype($output) === "string" ? array("error" => $output) : $output;
+        // set content type to JSON
+        header('Content-Type: application/json');
+        // echo JSON error message
+        echo json_encode($data);
+        exit;
+    }
+
+    if ($requestedType === "png") {
+        echoAsPng($card);
+        exit;
+    }
+    
+    echoAsSvg($card);
```

Comment:
This should exit as well since if a card is rendered at any time in the program, it should not continue with rendering.

```suggestion
    echoAsSvg($card);
    exit;
```